SETUP.md

```diff
@@ -693,7 +693,7 @@ dashboard.example.com_postfix_1       /sbin/my_init                    Up
 - In the Host machine, a cronjob needs to be configured as below.
 
     ```console
-    sudo echo "0 * * * * /bin/bash -l -c '/opt/docker/dashboard.example.com/apiserver/version_check.sh'" >> /etc/crontab
+    sudo echo "0  *    * * *   root    /bin/bash -l -c '/opt/docker/training.mcci.io/apiserver/version_check.sh'" >> /etc/crontab
     ```
```

Comment:
In addition: assets/postfix_node_1.png, assets/postfix_node_2.png, assets/postfix_node_3.png, and the other files use "demo.com", "demi.com", etc.

Must always use "consistent name", in this case "`dashboard.example.com`" is what I suggest.

Fix them all, please.

Reply:
yes Terry, it's a bug, will correct it

Reply:
updated with "user@example.com/test@example.com" for email Ids and "dashboard.example.com" for dashboard URLs.